File: Tests/CountriesFlagsObjCTests/EnvironmentTests.m
//
//  EnvironmentTests.m
//  CountriesFlags
//

#import <XCTest/XCTest.h>

@interface EnvironmentTests : XCTestCase

@end

@implementation EnvironmentTests

- (void)setUp {
    // Put setup code here. This method is called before the invocation of each test method in the class.
}

- (void)tearDown {
    // Put teardown code here. This method is called after the invocation of each test method in the class.
}

- (void)testScreenScaleFactor {
    XCTAssertTrue([[UIScreen mainScreen] scale] == 2.0, "All tests must be run on device with @2x screen scale factor");
}

@end

File: Tests/CountriesFlagsTests/EnvironmentTests.swift
//
//  EnvironmentTests.swift
//  
//  EnvironmentTests.swift
//

import XCTest

final class EnvironmentTests: XCTestCase {

//    override func setUpWithError() throws {
//        // Put setup code here. This method is called before the invocation of each test method in the class.
//    }

//    override func tearDownWithError() throws {
//        // Put teardown code here. This method is called after the invocation of each test method in the class.
//    }

    func testScreenScaleFactor() throws {
        XCTAssertTrue(UIScreen.main.scale == 2.0, "All tests must be run on device with @2x screen scale factor")
    }
}
